EIPS/eip-ase.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7676
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.